dask/dataframe/tests/test_reshape.py

```diff
@@ -170,28 +170,30 @@ def test_get_dummies_errors():
         dd.get_dummies(ddf.x)
 
 
+@pytest.mark.parametrize("values", ["B", ["B"], ["B", "D"]])
```

Comment:
Still accepts a scalar value like `"B"`?

Reply:
yeah -- just to be backwards compatible and to be closer to parity w/Pandas which accepts scalar values and list.

Reply:
:+1: